app/components/layout/FindHeader.jsx

```diff
@@ -33,7 +42,7 @@ class FindHeader extends React.Component {
             ref={(c) => { this.searchComponent = c; }}
             type="text"
             className="search-field"
-            placeholder='try "rental assistance" or "Compass Family Shelter"'
+            placeholder="Search 1,768 resources in San Francisco"
```

Comment:
NIT: It feels bad to hardcode this number. Would "over 1,700" work or something more general, @derekfidler?

Reply:
Oh, you left that in the description above. 🤦‍♂️ I apologize for not reading.